.github/workflows/test_and_deploy.yml

```diff
@@ -10,14 +10,18 @@ jobs:
 
   test:
     needs: lint
+    name: ${{ matrix.os }} py${{ matrix.python-version }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9]
-        exclude:
-          - {os: "windows-latest", python-version: "3.7"}
-          - {os: "windows-latest", python-version: "3.8"}
+        # Run all supported Python versions on linux
+        python-version: ["3.7", "3.8", "3.9"]
+        os: [ubuntu-latest]
+        # Include one windows run
+        include:
+        - os: windows-latest
+          python-version: "3.9"
```

Comment:
I know it's UNIX anyway, but should we be testing on macOS?

Reply:
We haven't been because distributing NiftyReg macOS binaries was difficult, and so not officially supported. Now NiftyReg is on conda, we could add the tests back, but we would need to add a NiftyReg installation step.

Reply:
I've raised #77 to track macOS support. For now we can leave the testing as it is.